app/reducers/pageReducer.ts

```diff
@@ -98,18 +109,24 @@ const pageMenuItemReducer = (state: PageDefinition, action: AnyAction) => {
         case types.REMOVE_LAYOUT_SECTION:
             return { ...state, layout: _.without(state.layout, _.nth(state.layout, action.layoutSection)) };
         case types.ADD_DRILLDOWN_PAGE: {
-            const pageData = _.cloneDeep(state);
-            forAllWidgets(pageData, layoutSectionWidgets => widgets(layoutSectionWidgets, action));
+            let pageMenuItem = _.cloneDeep(state);
+
+            // Update widget that created drilldown page
+            const pagesList: PageDefinition[] = pageMenuItem.type === 'pageGroup' ? pageMenuItem.pages : [pageMenuItem];
+            _.each(pagesList, p => forAllWidgets(p, layoutSectionWidgets => widgets(layoutSectionWidgets, action)));
```

Comment:
WDYT about ommiting acronyms? 🤔 

```suggestion
            _.each(pagesList, page =>
                forAllWidgets(page, layoutSectionWidgets => widgets(layoutSectionWidgets, action))
            );
```

Reply:
Sure, will apply it.